### DISCUSSION TITLE
Hyprland, Arch Linux and the AUR - support for the maintainers

### DISCUSSION THREAD

**Discussion by arcolinuxz** [hyprwm/Hyprland#1787] [General]
For the future - could you keep an eye on the comments of your package on the AUR of Arch Linux?
We seem to have an issue with the latest version 0.23 on just one package.

Could you please assist the maintainers of the PKGBUILDS for hyprland when you are making changes to your project?

**Unsuccessful builds for**

https://aur.archlinux.org/packages/hyprland

error code
`ninja: error: '/home/erik/ARCO/ARCOLINUX-PKGBUILD/arcolinux-pkgbuild-3party/ArcoLinux_3th_party/hyprland/src/hyprland-source/subprojects/udis86/build/libudis86/liblibudis86.a', needed by 'Hyprland', missing and no known rule to make it
make: *** [Makefile:154: release] Error 1
==> ERROR: A failure occurred in build().`

**Successful builds for** 

https://aur.archlinux.org/packages/hyprland-nvidia
hyprland-nvidia-0.23.0beta-1-any.pkg.tar.zst

and

https://aur.archlinux.org/packages/hyprland-nvidia-git
hyprland-nvidia-git-r2592.569ae86c-1-any.pkg.tar.zst

and

https://aur.archlinux.org/packages/hyprland-git
hyprland-git-r2592.569ae86c-1

**Comment by vaxerski** [comment]
Fixed. Well sorry for me not being awake at

![image](https://user-images.githubusercontent.com/43317083/225332615-df11cee0-b867-4572-9984-7593d0d89527.png)

I woke up at 1:30PM today and fixed it within 10 mins. Jeez.

> Could you please assist the maintainers of the PKGBUILDS for hyprland when you are making changes to your project?

I am the maintainer of hyprland and hyprland-bin. I just made a typo. @ThatOneCalculator is either asleep for days at once or on a road trip to kentucky usually so I update those most of the time

On that note kainoa next time you make a road trip to kentucky get me some kfc bby <3

**Reply by arcolinuxz** [reply]
no need to apologize
we are very very grateful

**Reply by ThatOneCalculator** [reply]
Sorry about that. I've just been really busy irl, coming off of a trip and then switching between jobs.
